lib/pine.coffee

```diff
@@ -11,6 +11,10 @@ class ResinPine extends PinejsClientCore
 		params.json = true
 		params.gzip ?= true
 
+		# Support "data" attribute
+		if params.data? and not params.body?
```

Comment:
True, I tested this branch on the SDK and `data` is not even passed, therefore this PR is irrelevant. I'll close for now.